### ISSUE TITLE
abstol is missing?

### ISSUE THREAD

**Issue by ChrisRackauckas** [JuliaMath/Roots.jl#126]
```julia
  Got exception outside of a @test
  LoadError: MethodError: no method matching init_options(::Roots.FalsePosition{:anderson_bjork}, ::Roots.UnivariateZeroState{Float64,Float64}; abstol=2.220446049250313e-16)
  Closest candidates are:
    init_options(::Roots.AbstractUnivariateZeroMethod, ::Roots.UnivariateZeroState{T,S}; xatol, xrtol, atol, rtol, maxevals, maxfnevals, strict) where {T, S} at C:\Users\Chris Rackauckas\.julia\packages\Roots\p2CdN\src\find_zero.jl:137 got unsupported keyword argument "abstol"
    init_options(!Matched::M<:Union{Bisection, BisectionExact}, ::Roots.UnivariateZeroState{T,S}; xatol, xrtol, atol, rtol, maxevals, maxfnevals) where {M<:Union{Bisection, BisectionExact}, T, S} at C:\Users\Chris Rackauckas\.julia\packages\Roots\p2CdN\src\bracketing.jl:225 got unsupported keyword argument "abstol"
    init_options(!Matched::Roots.AbstractAlefeldPotraShi, ::Roots.UnivariateZeroState{T,S}; xatol, xrtol, atol, rtol, maxevals, maxfnevals) where {T, S} at C:\Users\Chris Rackauckas\.julia\packages\Roots\p2CdN\src\bracketing.jl:564 got unsupported keyword argument "abstol"
    ...
  Stacktrace:
   [1] kwerr(::NamedTuple{(:abstol,),Tuple{Float64}}, ::Function, ::Roots.FalsePosition{:anderson_bjork}, ::Roots.UnivariateZeroState{Float64,Float64}) at .\error.jl:97
```

**Comment by jverzani** [comment]
Is this in a test? In the last tagged version I removed the code that renamed the tolerance arguments (the old `abstol`-> `atol`) (they are similar to `isapprox` with `xatol`, `xrtol`, `atol`, and `rtol`). So, that should be `xatol` now, I think. Sorry, I failed to note that. Will this be an issue? I could add it back in.

On a related note, I redid the `A42` method, which had poor performance, to be on par with the `FalsePosition` methods and added two others. Now, there are methods `Roots.A42`, `Roots.AlefeldPotraShi`, and `Roots.Brent` to do bracketed zero finding that are similar to FalsePosition, but more robust, as seen on the "RootTesting" tests. `Brent` is faster for cheaply computed functions, `A42` uses fewer function calls.
